### DISCUSSION TITLE
How to make NotepadNext compatible with NVDA

### DISCUSSION THREAD

**Discussion by cary-rowen** [nvaccess/nvda#14604] [Q&A]
[NotepadNext](https://github.com/dail8859/NotepadNext): A cross-platform, reimplementation of Notepad++
The program's menu bar is compatible with NVDA just fine, but the editable controls don't interact with NVDA, which I'm not familiar with since it's built using QT.
I commited an Issue at https://github.com/dail8859/NotepadNext/issues/309 but the author is not familiar with accessibility.
Anyone in our NVDA community, any suggestions on this?

Thanks

**Comment by Brian1Gaff** [comment]
That is a big shame, I was given to understand that many programmers are now 
taught about accessibility, but obviously the existing ones if they want to 
get into work will have to learn about it, as the USA is a big market and 
since all software bought by anything to do with government has got to be 
accessible it makes sense.Don't want another quick books cock up.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message ----- 
From: "Rowen" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Tuesday, January 31, 2023 10:29 PM
Subject: [nvaccess/nvda] How to make NotepadNext compatible with NVDA 
(Discussion #14604)
